setup.py

```diff
@@ -32,7 +32,7 @@
         'names',
         'pyasn1',
         'pycryptodome',
-        'pyopenssl',
+        'pyopenssl==19.1',
```

Comment:
They seem to be careful about backward-incompatible changes. Maybe we should pin the major version only, not the whole thing so we would get eventual security fixes?

Reply:
I locked to the minor version `.1`, so I thought it was enough to receive the security updates, but it made me realise that they may not the older minor versions. I'll lock to major.